src/EventListener/MergeHttpHeadersListener.php

```diff
@@ -92,8 +92,10 @@ private function mergeHttpHeaders(Response $response)
                 header_remove($name);
             }
 
-            // Do not replace existing headers as the response object has a higher priority
-            $response->headers->set($name, trim($content), false);
```

Comment:
Why is this necessary? The third argument should prevent replacing existing headers already.